src/components/templates/thumbnails/HoverDissolveThumbnail.vue

```diff
@@ -32,14 +40,15 @@ const isVideoType =
   false
 
 const baseImageClass = computed(() => {
-  return `absolute inset-0 ${isVideoType ? 'w-full h-full object-cover' : 'max-w-full max-h-64 object-contain'}`
+  const sizeClasses = isVideoType
+    ? 'w-full h-full object-cover'
```

Comment:
I think these could be `size-full`.
Also, if the PR that adds clsx and tailwind merge is in, you could use the cn function here.